sorbet/rbi/todo.rbi

```diff
@@ -76,3 +76,14 @@ module T::Private::Methods::MethodHooks; end
 module T::Private::Methods::Signature; end
 module T::Private::Methods::SingletonMethodHooks; end
 module T::Props::Plugin::ClassMethods; end
+
+class ActiveModel::Validations::PresenceValidator
```

Comment:
you can add this to the rbi in dependent_gem_rbis?
https://github.com/chanzuckerberg/sorbet-rails/blob/master/lib/sorbet-rails/dependent_gem_rbis/activerecord.rbi
(that file or you can create activemodel.rbi)

Reply:
This file may be overridden by sorbet when we regenerate rbi for new sorbet versions